src/core.c

```diff
@@ -277,10 +278,15 @@ ROX_INTERNAL RoxStateCode rox_core_setup(
             if (!api_key || !api_key[0]) {
                 ROX_ERROR("Invalid rollout apikey - must be specified");
                 return RoxErrorEmptyApiKey;
-            } else if (!str_matches(api_key, mongoIdPattern, PCRE2_CASELESS) && !str_matches(api_key, mongoIdPattern, PCRE2_CASELESS)) {
+            } else if (!str_matches(api_key, mongoIdPattern, PCRE2_CASELESS) && !str_matches(api_key, uuidIdPattern, PCRE2_CASELESS)) {
```

Comment:
using `str_matches(api_key, uuidIdPattern, PCRE2_CASELESS)` twice
can reuse

Reply:
I can throw it into a method.  I figured this was temporary anyway as eventually this would be refactored to just being a check for the platform key.

Reply:
can just put it in a variable, no need for a method, yet.. 😄